lib/building_oregon/field_sanitizer.rb

```diff
@@ -0,0 +1,28 @@
+module BuildingOregon
+  class FieldSanitizer
+    attr_reader :field
+    def initialize(field)
+      @field = field.dup
+    end
+
+    def sanitize
+      remove_metadata_tag
+    end
+
+    private
+
+    def remove_metadata_tag
+      @field.gsub!("desc_metadata__", "")
+      remove_format
+    end
+
+    def remove_format
+      format_list.each {|format| @field.gsub!(format, "")}
+      @field
```

Comment:
I didn't really know where to put this, since I want the field to be returned instead of the format_list. I felt like I could have either put it into the sanitize field, or where it is currently? Any recommendations?